tezedge_state/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "tezedge-state"
+version = "0.1.0"
```

Comment:
@binier 
this will be 1.7.0,
but I have fixed it in consequent PR:
https://github.com/tezedge/tezedge/pull/752/files